### PR TITLE
Milestone3 wmd

### PR DESCRIPTION
**Pull request by crystaljesica**
new features:
add translation
fix the bug of testing
update the css style

### CODE REVIEW

File: src/app/demo/demo.component.ts [hunk 1]
```diff
@@ -27,7 +28,71 @@ export class DemoComponent implements OnInit {
...
+    }
+    const ss = doc.querySelectorAll("s");
+    let count = 0;
+    ss.forEach((tag_s) => {
+      tag_s.insertAdjacentHTML(
+        "afterend",
+        `<span class="translation" contenteditable="true">${translation[count].innerHTML}</span>`
+      );
+      count++;
+    });
```
Comment (yoxchen95) [inline src/app/demo/demo.component.ts:82]: It is also okay use span to insert the new translation line into the xml file and make html visible to that change. But xiaotong seems to add the similar changes about this function.

File: src/app/demo/demo.component.ts [hunk 2]
```diff
@@ -27,7 +28,71 @@ export class DemoComponent implements OnInit {
...
+      innerbutton.addEventListener("click", () => {
+        sentence.insertAdjacentHTML(
+          "beforeend",
+          '<br><span class = "translation" contenteditable = True>Translation</span>'
+        );
+        innerbutton.disabled = true;
+      });
+      sentence.insertAdjacentElement("afterend", innerbutton);
+    });
+  }
```
Comment (yoxchen95) [inline src/app/demo/demo.component.ts:52]: This function works well for the add translation line. It is good.

Comment (crystaljesica) [reply src/app/demo/demo.component.ts:52]: Thanks for review.

File: src/app/demo/demo.component.ts (hunk 1 above)
Comment (crystaljesica) [reply src/app/demo/demo.component.ts:82]: Combine the milestone3 together of one brunch

**Comment by crystaljesica** [comment]
Since already combine all the feature into one brunch and modified the bugs, I will close this pull request. This is same to the previous pull request which has already been merged.

**Pull request closed without merging**